File: bankomat/src/com/adrian/entity/Account.java
package com.adrian.entity;

import java.util.Scanner;

public class Account {

    private double balance = 5000;

    public void deposit(Scanner input) {

        System.out.print("Ile pieniędzy chcesz wpłacić na konto: ");
        double money = input.nextDouble();

        balance += money;

        System.out.println("Pieniądze zostąły wpłacone!");
    }

    public void withdrawal(Scanner input) {

        System.out.print("Ile pieniędzy chcesz wypłacić: ");
        double money = input.nextDouble();

        if (money > balance) System.out.println("Brak wystarczających środków na koncie !!!");
        else balance -= money;
    }

    public double getBalance() {
        return balance;
    }

    public void setBalance(double balance) {
        this.balance = balance;
    }
}

File: APO/Account.py
class Account:
	def __init__(self, id, balance=0,):
		self.id = id
		self.balance = balance

	def getId(self):
		return self.id

	def getBalance(self):
		return self.balance

	def withdraw(self, amount):
		self.balance -= amount

	def deposit(self, amount):
		self.balance += amount
